patches/server/0943-Adding-config-to-allow-marker-ticking.patch

```diff
@@ -0,0 +1,61 @@
++                if (!chunkProviderServer.isPositionTicking(e) || (e instanceof net.minecraft.world.entity.Marker && !world.paperConfig().entities.tickMarkers)) { // Paper - Markers aren't ticked by default.
++                	nonEntityTicking.merge(key, 1, Integer::sum);
+                 }
++                
+             });
+             if (names.size() == 1) {
+                 ResourceLocation name = names.iterator().next();
+diff --git a/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java b/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
+index e2c612dd55fcb2769fb06f7878b8d0873f2be139..e5543c1123073323cb9071da2c826d3b238ef84e 100644
+--- a/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/WorldConfiguration.java
+@@ -98,6 +98,7 @@ public class WorldConfiguration extends ConfigurationPart {
+ 
+     public class Entities extends ConfigurationPart {
+         public boolean entitiesTargetWithFollowRange = false;
++        public boolean tickMarkers = false;
+         public MobEffects mobEffects;
+ 
+         public class MobEffects extends ConfigurationPart {
+diff --git a/src/main/java/net/minecraft/server/level/ServerLevel.java b/src/main/java/net/minecraft/server/level/ServerLevel.java
+index 96bb0e56f12437037b598cd7baabf369e5994517..207ea012f0afcd28c35d5d393a681a3e21ef60db 100644
+--- a/src/main/java/net/minecraft/server/level/ServerLevel.java
++++ b/src/main/java/net/minecraft/server/level/ServerLevel.java
+@@ -2504,7 +2504,7 @@ public class ServerLevel extends Level implements WorldGenLevel {
+         }
+ 
+         public void onTickingStart(Entity entity) {
+-            if (entity instanceof net.minecraft.world.entity.Marker) return; // Paper - Don't tick markers
++            if (entity instanceof net.minecraft.world.entity.Marker && !paperConfig().entities.tickMarkers) return; // Paper - Don't tick markers by default
+             ServerLevel.this.entityTickList.add(entity);
+             ServerLevel.this.entityManager.addNavigatorsIfPathingToRegion(entity); // Paper - optimise notify
+         }
+diff --git a/src/main/java/org/spigotmc/ActivationRange.java b/src/main/java/org/spigotmc/ActivationRange.java
+index 40b382c2e0e33fe5c24a51b211cd2f9557a60c5e..fb8bed037809040003a12553ac704a0483bfb232 100644
+--- a/src/main/java/org/spigotmc/ActivationRange.java
++++ b/src/main/java/org/spigotmc/ActivationRange.java
+@@ -212,7 +212,7 @@ public class ActivationRange
+             // Paper end
+ 
+             // Paper start
+-            java.util.List<Entity> entities = world.getEntities((Entity)null, maxBB, (e) -> !(e instanceof net.minecraft.world.entity.Marker)); // Don't tick markers
++            java.util.List<Entity> entities = world.getEntities((Entity)null, maxBB, (e) -> (e instanceof net.minecraft.world.entity.Marker && world.paperConfig().entities.tickMarkers)); // Don't tick markers by default
```

Comment:
This will actually just allow marker to be activated or not but all others entities will be ignored. And the old behavior activate all entities except markers
should probably be:
!(e instanceof net.minecraft.world.entity.Marker) || tickMarkers ?